test/sandbox/test/base_test.rb

```diff
@@ -154,4 +154,21 @@ def test_no_method_error_does_not_reference_missing_helper
     MESSAGE
     assert !exception_message_regex.match?(exception.message)
   end
+
+  def test_strict_helpers_enabled
+    with_config_option(:strict_helpers_enabled, false) do
+      top_level_component_class = Class.new(ViewComponent::Base)
+      refute ViewComponent::Base.config.strict_helpers_enabled?, ".strict_helpers_enabled? should be false by default"
+      # refute ViewComponent::Base.new.config.strict_helpers_enabled?, "#strict_helpers_enabled? should be false by default"
```

Comment:
Previously I was thinking `ViewComponent::Base` might have a `config` when instantiated, but that's probably not likely to be the case.

There's perhaps also a clash with `use_helper :config` that's in there by default.

Reply:
Moved component-level config to `component_config`. `ViewComponent::Base` itself still responds on `config`.